docs/dns-usage.rst

```diff
@@ -0,0 +1,191 @@
+Using the API
+=============
+
+Client
+------
+
+:class:`Client <gcloud.dns.client.Client>` objects provide a means
+configure your DNS applications.  Eash instance holds both a ``project``
+and an authenticated connection to the DNS service.
+
+For an overview of authentication in ``gcloud-python``, see:
+http://gcloud-python.readthedocs.org/en/latest/gcloud-auth.html
```

Comment:
We should use Sphinx to just link to the `gcloud-auth` doc (not specific to the domain, and not default to `http` over `https`)

How about:

```
see the `Authentication doc`_.

.. _Authentication doc: gcloud-auth.html
```

Sphinx also has a `:doc:` directive but it ends up italicizing the link, which looks different than the rest.